src/status_im/ui2/screens/chat/photo_selector/style.cljs

```diff
@@ -0,0 +1,73 @@
+(ns status-im.ui2.screens.chat.photo-selector.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(defn remove-photo-container []
+  {:width            14
+   :height           14
+   :border-radius    7
+   :background-color colors/neutral-50
+   :position         :absolute
+   :top              -7
+   :right            -7
+   :justify-content  :center
+   :align-items      :center})
+
+(defn gradient-container [safe-area]
+  {:width    "100%"
+   :height   (+ (:bottom safe-area) 161)
+   :position :absolute
+   :bottom   0})
+
+(defn buttons-container [safe-area]
+  {:flex-direction     :row
+   :justify-content    :space-between
+   :padding-horizontal 20
+   :bottom             (+ (:bottom safe-area) 33)})
+
+(defn clear-container []
+  {:background-color   (colors/theme-colors colors/neutral-10 colors/neutral-80)
+   :position           :absolute
+   :align-self         :flex-end
+   :padding-horizontal 12
+   :padding-vertical   5
+   :right              20
+   :border-radius      10})
+
+(defn camera-button-container []
+  {:background-color (colors/theme-colors colors/neutral-10 colors/neutral-80)
+   :width            32
+   :height           32
+   :border-radius    10
+   :justify-content  :center
+   :align-items      :center
+   :margin-left      20
+   :margin-bottom    24})
+
+(defn chevron-container []
+  {:background-color (colors/theme-colors colors/neutral-10 colors/neutral-80)
+   :width            14
+   :height           14
+   :border-radius    7
+   :justify-content  :center
+   :align-items      :center
+   :margin-left      7
+   :margin-top       4})
```

Comment:
Can be `def,` since it doesn't need any parameters and will always be the same

Reply:
This is a very common source of misconception, the map actually needs to be dynamically allocated because it relies on the current value of a deref'ed atom (reactive or not it doesn't matter).

You can see this recent thread https://github.com/status-im/status-mobile/pull/14419#discussion_r1029715208, but I've commented about this a handful of times already in other PRs. I'm already thinking about creating a custom linter rule for this because it's very easy to miss for both reviewers and implementers.

Reply:
Learnt something new today
Thanks Icaro <3